test/support/api_response_tests.ex

```diff
@@ -62,6 +62,14 @@ defmodule ApiResponseTests do
           assert response == unquote(success_data)
         end
       end
+
+      test "it adds options to URL if it exists" do
+        Application.put_env(:google_geocoding_api, :region, "XYZ")
```

Comment:
We shouldn't need the env variable right?

Reply:
Oops, forgot to remove that part.